frontend/src/pages/PipelineDetailsV2.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+import { CommonTestWrapper } from 'src/TestWrapper';
+import { testBestPractices } from '../TestUtils';
+import PipelineDetailsV2 from './PipelineDetailsV2';
+
+testBestPractices();
+describe('PipelineDetailsV2', () => {
+  beforeEach(() => {
+    // Required by reactflow render.
```

Comment:
Could this be considered react-flow bug that we should report to them?

Reply:
That is a good point, it looks like that it is caused by the combination of using Create React App and react testing library, reference: https://stackoverflow.com/questions/64558062/how-to-mock-resizeobserver-to-work-in-unit-tests-using-react-testing-library. So it might not be something reactflow library can help here.